content/en/docs/ops/configuration/security/migrate-alpha-policy/index.md

```diff
@@ -0,0 +1,338 @@
+---
+title: Migrate the deprecated alpha security policy
+description: A tutorial to help customers to migrate the deprecated alpha security policy to the beta version.
+keywords: [security,policy,migrate,alpha,beta,deprecate,peer,jwt,authorization]
+weight: 20
+owner: istio/wg-security-maintainers
+test: n/a
+---
+
+Istio beta security policy (`PeerAuthentication`, `RequestAuthentication` and `AuthorizationPolicy`) has been released
+since release 1.5 and the alpha security policy (`MeshPolicy`, `Policy`, `ClusterRbacConfig`, `ServiceRole` and `ServiceRoleBinding`)
+has been deprecated and no longer supported starting 1.6.
+
+It is required to migrate the alpha security policy to the beta version before upgrading to Istio 1.6 and later.
+This blog is a tutorial to help customers who are still using the alpha security policy in the old version to migrate to
+the beta security policy and unblock the upgrade to Istio 1.6 and later.
+
+## Overview
+
+The control plane should be upgraded first to bring in the support of `v1beta1` security policy. It is recommended first
+upgrade to Istio 1.5 as a transitive version because it supports both `v1alpha1` and `v1beta1` security policy. You complete
+the security policy migration in Istio 1.5, remove the `v1alpha1` security policy and then continue the upgrade to later Istio versions.
+
+Istio 1.5 is the only version that supports both `v1alpha1` and `v1beta1` security policy. For a given workload, the
+`v1beta1` version will take precedence over the `v1alpha1` version.
+
+Alternatively, if you want to do a skip-upgrade directly from Istio 1.4 to Istio 1.6 or later, you should use the
+[canary upgrade](/docs/setup/upgrade/canary/) to install the new Istio version as a separate control plane, gradually
+migrate your workloads to the new control plane and complete the security policy migration at the same time.
+
+Note that skip-upgrade is not supported by Istio and there might be other issues in this process. Istio 1.6 does not support
```

Comment:
@jacob-delgado @Monkeyanator Anything to add on this first sentence about skip upgrades?

Reply:
Should we even mention the skip-level upgrade as an alternative here if it's not supported? I'm fine with it either way since we do mention that it's dangerous

Reply:
We know some customers are actively considering the skip-upgrade option, so just want to keep the information and warning here in case others might want to do it.